app/models/letter_rates.py

```diff
@@ -35,7 +35,10 @@ class LetterRates(ModelList):
 
     @staticmethod
     def _get_items(*args, **kwargs):
-        return letter_rate_api_client.get_letter_rates(*args, **kwargs)
+        # Once the economy postage is live, we can remove this logic
+        resp = letter_rate_api_client.get_letter_rates(*args, **kwargs)
+        filtered_letter_rates = filter(lambda r: r["post_class"] in LetterRates.post_classes.keys(), resp)
+        return list(filtered_letter_rates)
```

Comment:
- using a list comprehension is more Pythonic than a lambda
- `x in dict` is functionally the same as `x in dict.keys()` but a bit shorter
```suggestion
        return [rate for rate in resp if rate["post_class"] in LetterRates.post_classes]
```